modin/pandas/test/test_dataframe.py

```diff
@@ -2125,8 +2125,9 @@ def test_combine_first(self):
 
```

Comment:
We can parametrize this test since it is testing correctness now.

Reply:
I looked at `DataFrame.corr` from performance perspective. It looks very slowly now. After the test was parameterized containerized tests on Ray/Dask failed because of timeout. Also, some checks were cancelled. I think, it happens because of there are lots of operations that are executed remotely in the [inner loop](https://github.com/modin-project/modin/pull/1521/files#diff-73ebd3f1e2f845adf0a4281cb08bf6bcR2728). I don't think, it is possible to parallelize `DataFrame.corr` effectively considering missing values as well as pandas do it. May be, make sense to keep implementation that is defaults to pandas, what do you think?

Reply:
It was decided to merge `Series.corr` into master and think on implementation for `DataFrame.corr`.